### ISSUE TITLE
Azure AD, work or school, personal?

### ISSUE THREAD

**Issue by solidcloudio** [projectkudu/ARMExplorer#126]
My organizational account email (user@domain.com) was added as a subscription admin to a users Azure AD.   If I login to azure with my "personal account" (with the same email), I can see his azure resources.  If I'm logged in with my organizational account, I can't see the resources.

How did him using my email, associate the login to personal and not work or school.
Another customer did this last week and it worked the opposite way, I could see his resources with my organizational account, not my personal one.

**Comment by davidebbo** [comment]
The behavior should match what you see in portal.azure.com if you use the same account and Directory. Can you confirm?

**Comment by solidcloudio** [comment]
No, it doesn't work the same...  with the portal I can login 2 ways..  with ARM Explorer, its forcing me to login with my organizational account
